tests/assistant_spec.js

```diff
@@ -478,6 +479,182 @@ describe('TranslationAssistant (math)', function() {
+    });
+
+    it('should translate multiple math notations at once', function() {
+        const allItems = [];
+        const itemsToTranslate = [
+            {englishStr: '$3 \\mult x = 9.9 \\div 3$', translatedStr: ''},
+        ];
+        const translatedStrs = ['$3 \\cdot x = 9{,}9 \\mathbin{:} 3$'];
+
+        assertSuggestions(allItems, itemsToTranslate, translatedStrs, 'de');
+    });
+
+    it('should translate multiple decimals', function() {
+        const allItems = [];
+        const itemsToTranslate = [
+            {englishStr: '$3 \\mult x = 9.9 \\div 3.3$', translatedStr: ''},
+        ];
+        const translatedStrs = ['$3 \\cdot x = 9{,}9 \\mathbin{:} 3{,}3$'];
+
+        assertSuggestions(allItems, itemsToTranslate, translatedStrs, 'de');
+    });
+
+    it('should translate math with \\text', function() {
+        const allItems = [{
+            englishStr: 'simplify $\\text{red} = 5.1$',
+            translatedStr: 'simplifyz $\\text{rouge} = 5{,}1$',
+        }];
+        const itemsToTranslate = [{
+            englishStr: 'simplify $3 * \\text{red} = 20.4$',
+            translatedStr: '',
+        }];
+        const translatedStrs = ['simplifyz $3 * \\text{rouge} = 20{,}4$'];
+
+        assertSuggestions(allItems, itemsToTranslate, translatedStrs, 'fr');
+    });
+
+    it('should translate math with \\textbf', function() {
+        const allItems = [{
+            englishStr: 'simplify $\\textbf{red} = 5.1$',
+            translatedStr: 'simplifyz $\\textbf{rouge} = 5{,}1$',
+        }];
+        const itemsToTranslate = [{
+            englishStr: 'simplify $3 * \\textbf{red} = 20.4$',
+            translatedStr: '',
+        }];
+        const translatedStrs = ['simplifyz $3 * \\textbf{rouge} = 20{,}4$'];
+
+        assertSuggestions(allItems, itemsToTranslate, translatedStrs, 'fr');
+    });
+
+    it('should translate math with \\text{} and \\textbf{}', function() {
+        const allItems = [{
+            englishStr: 'simplify $\\text{red} = 5 \\mult \\textbf{blue}$',
+            translatedStr: 'simplifyz $\\text{azul} = 5 \\cdot \\textbf{roja}$',
+        }];
+        const itemsToTranslate = [{
+            englishStr: 'simplify $3 \\text{red} = 2 \\div \\textbf{blue}$',
+            translatedStr: '',
+        }, {
+            englishStr: 'simplify $3 \\textbf{red} = 2 \\div \\text{blue}$',
+            translatedStr: '',
+        }];
+
+        // Even though "red" in Spanish should be "roja", smart translations
+        // doesn't know that.  The template built from allItems will contain a
+        // mathDictionary which maps "red" to "azul" and "blue" to "roja".
+        assertSuggestions(allItems, itemsToTranslate, [
+            'simplifyz $3 \\text{azul} = 2 \\mathbin{:} \\textbf{roja}$',
+            'simplifyz $3 \\textbf{azul} = 2 \\mathbin{:} \\text{roja}$',
+        ], 'cs');
+    });
+
+    // TODO(danielhollas): This we need to decide
```

Comment:
Do you wish to come to a decision before merging this pull request?

If so maybe ping Caroline in JIRA? Otherwise, if you're ok addressing this at a later point, as a separate change, lets expand the comment a bit to provide more context.

Reply:
I've given it a lot of thought and feel pretty good about the current behavior. I will delete this TODO and only leave a NOTE in the source. After this is deployed, I'll ask the i18n community for feedback and change this behavior if needed.